gitbootcamp.rst

```diff
@@ -480,10 +469,10 @@ To edit an open pull request that targets ``master``:
       git fetch upstream
       git merge upstream/master
       git add <filename>
-      git commit -m "<commit message>"
+      git commit -m <message>
```

Comment:
Same comment above. I think that the double quote is necessary because when you wirte
a commit message, tue ```"```` is necessary

Reply:
Agreed, but since both the single quote ``'`` and double quote ``"`` are both acceptable, we should use the single quote since it was used in the previous one for consistency.

Reply:
Okay, but let's use the double quotes as they allow the apostrophe punctuation mark in the commit message, and as they were already used in the two `git commit` commands of this document. Done.